common/lib/xmodule/xmodule/partitions/tests/test_partitions.py

```diff
@@ -509,10 +530,59 @@ def test_get_group(self):
 
         # assign first group and verify that it is returned for the user
         self.user_partition.scheme.current_group = groups[0]
-        group1 = self.partition_service.get_group(self.user_partition)
+        group1 = self.partition_service.get_group(self.user, self.user_partition)
         self.assertEqual(group1, groups[0])
 
         # switch to the second group and verify that it is returned for the user
         self.user_partition.scheme.current_group = groups[1]
-        group2 = self.partition_service.get_group(self.user_partition)
+        group2 = self.partition_service.get_group(self.user, self.user_partition)
         self.assertEqual(group2, groups[1])
+
+
+class TestGetCourseUserPartitions(PartitionServiceBaseClass):
+    """
+    Test the helper method get_course_user_partitions.
+    """
+
+    def setUp(self):
+        super(TestGetCourseUserPartitions, self).setUp()
+        # django.conf.settings is not available when nosetests are run
+        FEATURES['ENABLE_ENROLLMENT_TRACK_USER_PARTITION'] = True
```

Comment:
nit: Define 'ENABLE_ENROLLMENT_TRACK_USER_PARTITION' as a static because it is used more than once.